charts/seed-terraformer/charts/gcp-infra/templates/_main.tf

```diff
@@ -32,6 +32,14 @@ resource "google_compute_subnetwork" "subnetwork-nodes" {
   region        = "{{ required "google.region is required" .Values.google.region }}"
 }
 
+{{ if .Values.networks.internal -}}
```

Comment:
Please add `.networks.internal` to the example `values.yaml` of this chart as a reference